utils.go

```diff
@@ -66,26 +68,44 @@ func getQuerySnippet(req *http.Request) string {
 
 	u := getDecompressor(req)
 	if u == nil {
-		return data
+		if len(query) != 0 && len(data) != 0 {
```

Comment:
I think these 3 similar checks may be simplified. Consider to split the function into two:
```
func getQuerySnippet(req *http.Request) string {
	query := req.URL.Query().Get("query")
	body, err := getQuerySnippetFromBody(req)
	if len(query) != 0 && len(data) != 0 {
		query += "\n"
	}
	return query + body
```

Reply:
Thank you for your comment. I've fixed